src/modules/nathelper/nathelper.c

```diff
@@ -704,10 +707,12 @@ static int fix_nated_contact_f(struct sip_msg *msg, char *str1, char *str2)
  * Replaces ip:port pair in the Contact: field with the source address
  * of the packet.
  */
-static int set_contact_alias(struct sip_msg *msg)
+static int set_contact_alias(struct sip_msg *msg, int trim)
```

Comment:
By changing the signature of `set_contact_alias()` you broke the kemi export of it.

Reply:
> By changing the signature of `set_contact_alias()` you broke the kemi export of it.

sorry i haven't worked with the kemi export code yet. i pushed another commit which i _think_ fixes it